src/extensions/renderer/canvas/drawing-edges.js

```diff
@@ -208,8 +208,18 @@ CRp.drawEdgePath = function( edge, context, pts, type ){
       case 'straight':
       case 'segments':
       case 'haystack':
-        for( let i = 2; i + 1 < pts.length; i += 2 ){
-          context.lineTo( pts[ i ], pts[ i + 1] );
+        const taxiRadius= edge.pstyle( 'taxi-radius' )
+        const taxiRadiusValue = taxiRadius ? taxiRadius.value : 0
+
+        if( taxiRadiusValue > 0) {
+          for( let i = 2; i + 1 < pts.length; i += 2 ) {
+            context.arcTo( pts[ i - 2 ], pts[ i - 1 ], pts[ i ], pts[ i + 1 ], 2 * taxiRadiusValue )
+          }
+          context.lineTo( pts[ pts.length - 2 ], pts[ pts.length - 1 ] );
```

Comment:
It looks like the hit tests aren't working on the curves.  On the debug page (`npm run watch`):

https://github.com/cytoscape/cytoscape.js/assets/989043/9aed9253-c6da-4728-b5db-17471f60eb46

Reply:
The hit tests are here:

https://github.com/cytoscape/cytoscape.js/blob/a653d1b073bf65f24ae69db5927fc88492f34293/src/extensions/renderer/base/coord-ele-math/coords.js#L79-L318